packages/cli/test/fixtures/service/index.js

```diff
@@ -0,0 +1,61 @@
+const DATASOURCE_APP_PATH = 'src/datasources';
+const CONFIG_PATH = '.';
+const DUMMY_CONTENT = '--DUMMY VALUE--';
+
+exports.SANDBOX_FILES = [
+  {
+    path: CONFIG_PATH,
+    file: 'mysoapconfig.json',
+    content: `{
```

Comment:
Let's use `JSON.stringify` instead of a string template literal.

Ideally, the code creating these sandbox files should be smart enough to call `JSON.stringify(content, null, 2)` when `typeof content === 'object'`.